numpy/core/defchararray.py

```diff
@@ -835,6 +835,17 @@ def isdigit(a):
     See Also
     --------
     str.isdigit
+
```

Comment:
It looks like this change probably got committed on the wrong branch. I'm happy to help "fix" this if you'd like, just LMK!

Reply:
I have added examples of isdigit on another branch as well. Yeah, please fix it.